samples/model-builder/create_training_pipeline_custom_job_sample.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.cloud import aiplatform
+from typing import List
+
+
+#  [START aiplatform_sdk_create_training_pipeline_custom_job_sample]
+def create_training_pipeline_custom_job_sample(
+    project: str,
+    display_name: str,
+    args: List[str],
+    script_path: str,
+    container_uri: str,
+    location: str = "us-central1",
+    model_display_name: str = None,
+    training_fraction_split: float = 0.8,
```

Comment:
The following args are appropriate here as well to allow the user to customize their machines:
```
        replica_count: int = 0,
        machine_type: str = "n1-standard-4",
        accelerator_type: str = "ACCELERATOR_TYPE_UNSPECIFIED",
        accelerator_count: int = 0,
```

Reply:
Sure, will add.